stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  type: enum
+  label: Choose TLS Certificate Source
+  group: "Expose TLS Options"
+  options:
+  - "auto"
+  - "secret"
+  - "none"
+- variable: expose.tls.commonName
+  default: ""
+  required: true
+  description: "The common name used to generate the certificate"
+  type: string
+  label: Common Name for the TLS Certificate
+  show_if: "expose.tls.certSource=auto&&expose.type!=ingress"
+  group: "Expose TLS Options"
+- variable: expose.tls.auto.rotateCert
+  default: false
+  description: "Generate new TLS certificates during post-deployment configuration changes and upgrades"
+  type: boolean
+  label: Rotate TLS Certificates
+  show_if: "expose.tls.certSource=auto"
+  group: "Expose TLS Options"
+- variable: expose.tls.secret.secretName
+  default: ""
+  required: true
+  description: "The name of the Harbor TLS certificate secret. The secret must contain keys named tls.crt and tls.key representing the certificate and private key to use for TLS"
+  type: certificate
+  label: Select a Secret Holding the Harbor TLS Certificate
+  show_if: "expose.tls.certSource=secret"
+  group: "Expose TLS Options"
+- variable: expose.tls.secret.notarySecretName
+  default: ""
+  required: true
+  description: "The name of the Notary TLS certificate secret. The secret must contain keys named tls.crt and tls.key representing the certificate and private key to use for TLS"
+  type: certificate
+  label: Select a Secret Holding the Notary TLS Certificate
+  show_if: "expose.tls.certSource=secret&&expose.type=ingress"
+  group: "Expose TLS Options"
+- variable: internalTLS.enable
+  default: false
+  required: true
+  description: "Use TLS to secure internal communication between Harbor components"
+  type: boolean
+  label: Enable Internal TLS
+  group: "Internal TLS Options"
+- variable: internalTLS.rotateCert
+  default: false
+  description: "Generate new TLS certificates during post-deployment configuration changes and upgrades"
+  type: boolean
+  label: Rotate TLS Certificates
+  show_if: "internalTLS.enable=true"
+  group: "Internal TLS Options"
+# config imageChartStorage type
+- variable: persistence.imageChartStorage.type
+  default: "filesystem"
+  description: "Specify the type of storage for OCI artifacts: filesystem, azure, gcs, s3, swift, oss"
+  type: enum
+  label: OCI Artifact Storage Type
+  group: "OCI Artifact Storage Settings"
+  required: true
+  options:
+  - "filesystem"
+  - "azure"
+  - "gcs"
+  - "s3"
+  - "oss"
+  - "swift"
+- variable: persistence.imageChartStorage.disableredirect
+  default: false
+  description: "Specify whether to disable `redirect` for images and chart storage for backends which do not support it (such as using minio for `s3` storage type)"
+  type: boolean
+  label: Disable Redirect
+  group: "OCI Artifact Storage Settings"
+# azure storage
+- variable: persistence.imageChartStorage.azure.accountname
+  required: true
+  default: ""
+  description: "Azure account name"
```

Comment:
It's actually "Azure storage account name." (see the demo)

Reply:
fixed